keyboards/eason/aeroboard/aeroboard.h

```diff
@@ -0,0 +1,30 @@
+// Copyright 2022 Eason
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+
+ #pragma once
+
+ #include "quantum.h"
+
+ #define ___ KC_NO
+/* This is a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT_all( \
```

Comment:
(and everywhere else `LAYOUT_all` was used eg. `info.json` & keymaps)
```suggestion
#define LAYOUT( \
```